addon/components/menu.js

```diff
@@ -103,20 +103,20 @@ export default class Menu extends Component {
   }
 
   @action
-  registerItem(item) {
+  async registerItem(item) {
```

Comment:
long term, I think we need to find a different design for this -- this causes performance issues where we render _additional times_ for each item.

So, for 1000 items, we re-render 1000 times. not great. lol

(I assume Vue and React implementations have this same problem)